main.cpp

```diff
@@ -107,10 +107,54 @@ struct CarWash
     you should be able to deduce the return type of those functions based on their usage in Person::run()
     You'll need to insert the Person struct from the video in the space below.
  */
+struct Person
+{
+    int age;
+    int height;
+    float hairlength;
+    float GPA;
+    unsigned int SATScore;
+    int distanceTravelled;
+    int totalTime;
+
+    struct Foot
+    {
+    int strideDistance = 3;
+    int avgStepSize;
 
+    void stepForward();
+    int stepSize();
+    };
 
+    Foot leftFoot;
+    Foot rightFoot;
 
+    void run(int howFast, bool startWithLeftFoot);
+};
 
+void Person::Foot::stepForward()
+    {
+    }
+int Person::Foot::stepSize()
+    {
+        return avgStepSize;
+    }
```

Comment:
Fix this indentation
Curly braces should align with any keywords on previous lines
Code within a pair of curly braces should be indented 4 spaces relative to **those** curly braces.
- [ ] revise